pages/newcontext/base.py

```diff
@@ -21,3 +21,18 @@ def context(self):
         return dict(collections.ChainMap(
             *[ctx.context() for ctx in self.contexts]
         ))
+
+
+class SimpleContext(Context):
```

Comment:
Why do we need this? `Products` class does the same thing, but more elegantly

Reply:
sorry, forgot to rm it